steps/discord-message/0.0.1/step.yml

```diff
@@ -0,0 +1,59 @@
+title: Send a Discord message
+summary: |
+  Send a [Discord](https://discordapp.com) message to a text channel
+description: |
+  Send a [Discord](https://discordapp.com) message to a text channel using Discord Webhooks
+
+  To find your text channel **webhook** visit: [https://github.com/Akizo96/de.isekaidev.discord.wbbBridge/wiki/How-to-get-Webhook-ID-&-Token](https://github.com/Akizo96/de.isekaidev.discord.wbbBridge/wiki/How-to-get-Webhook-ID-&-Token)
+website: https://gitlab.com/lighthouseit/packages/bitrise-discord-message
+source_code_url: https://gitlab.com/lighthouseit/packages/bitrise-discord-message
+support_url: https://gitlab.com/lighthouseit/packages/bitrise-discord-message
```

Comment:
This should be https://gitlab.com/lighthouseit/packages/bitrise-discord-message/issues